pkg/resource/unstructured/composite/composite.go

```diff
@@ -60,110 +82,182 @@ func New(opts ...Option) *Unstructured {
 // +k8s:deepcopy-gen=true
 // +kubebuilder:object:root=true
 
-// An Unstructured composed resource.
+// An Unstructured composite resource.
 type Unstructured struct {
 	unstructured.Unstructured
+
+	Schema Schema
```

Comment:
So, if I got this correctly, the default schema is modern, for people upgrading that would mean that if they don't explicitly set the legacy one everything will be broken in a not so obvious way. Would it make sense to switch the two?